packages/react-components/react-combobox/src/components/Combobox/Combobox.types.ts

```diff
@@ -21,6 +22,11 @@ export type ComboboxSlots = {
 
   /* The dropdown listbox slot */
   listbox?: Slot<typeof Listbox>;
+
+  /* The dropdown portal slot.
+     Limited to setting props only as the listbox is the rendered child.
+  */
+  portal?: Slot<typeof Portal>;
```

Comment:
I think it'd make sense to have `portal` be non-optional and `NonNullable<>`, which then means you don't need the Omit/Pick below